app/lib/selections/tools/pen.dart

```diff
@@ -21,6 +21,29 @@ class PenToolSelection extends ToolSelection<PenTool> {
                       zoomDependent: value ?? selected.first.zoomDependent))
                   .toList())),
       const SizedBox(height: 16),
+      // ! new checkBox
+      CheckboxListTile(
+          value: selected.first.straightLineEnabled,
+          title: Text(AppLocalizations.of(context).straightLineEnabled),
+          onChanged: (value) => update(
+              context,
+              selected
+                  .map((e) => e.copyWith(
+                      straightLineEnabled: value ?? selected.first.straightLineEnabled))
+                  .toList())),
+      // ! new slider
```

Comment:
here too and maybe we should hide the slider if the checkbox is unchecked

Reply:
done! Do you think it can go?